internal/auth/options.go

```diff
@@ -3,310 +3,64 @@ package auth
 import (
 	"encoding/base64"
 	"fmt"
-	"net/http"
 	"net/url"
 	"os"
 	"path"
-	"reflect"
-	"strings"
-	"time"
 
 	"github.com/buzzfeed/sso/internal/auth/providers"
+	"github.com/buzzfeed/sso/internal/pkg/aead"
 	"github.com/buzzfeed/sso/internal/pkg/groups"
+	"github.com/buzzfeed/sso/internal/pkg/sessions"
 
 	"github.com/datadog/datadog-go/statsd"
-	"github.com/spf13/viper"
 )
 
-// Options are config options that can be set by environment variables
```

Comment:
I wonder what the best way is for us to capture this kind of information now? Could using something like godoc help with this, among other things?